src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -46,18 +46,19 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableUnsafeBinaryFormatterSerialization Condition="'$(EnableUnsafeBinaryFormatterSerialization)' == ''">false</EnableUnsafeBinaryFormatterSerialization>
   </PropertyGroup>
 
+  <!-- Default settings for trimmed and aot'd apps. -->
   <PropertyGroup Condition="'$(PublishTrimmed)' == 'true' Or '$(PublishAot)' == 'true'">
-    <!-- Runtime feature defaults to trim unnecessary code -->
     <JsonSerializerIsReflectionEnabledByDefault Condition="'$(JsonSerializerIsReflectionEnabledByDefault)' == ''">false</JsonSerializerIsReflectionEnabledByDefault>
   </PropertyGroup>
 
-  <!--
-    Enable the RequestDelegateGenerator by default for NativeAot apps.
-    This will be enabled for all PublishTrimmed apps once the RequestDelegateGenerator supports all MinimalApi features.
-   -->
+  <!-- Default settings for aot'd apps. -->
   <PropertyGroup Condition="'$(PublishAot)' == 'true'">
+    <!-- Enable the built-in source generators. -->
     <EnableRequestDelegateGenerator Condition="'$(EnableRequestDelegateGenerator)' == ''">true</EnableRequestDelegateGenerator>
     <EnableConfigurationBindingGenerator Condition="'$(EnableConfigurationBindingGenerator)' == ''">true</EnableConfigurationBindingGenerator>
+
+    <!-- Enable EventSource support because it is disabled by default in aot'd apps. -->
+    <EventSourceSupport Condition="'$(EventSourceSupport)' == ''">true</EventSourceSupport>
```

Comment:
Unfortunately this is too late. The NativeAOT targets have already defaulted this to `false` by now.

Reply:
Would somewhere around here work better?

https://github.com/dotnet/sdk/blob/4b18a3ca87269fce6b915586de2e7e7ae5d80e48/src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.targets#L97-L99

Reply:
Unfortunately that is still too late. (It is even later than where I put it above).

https://github.com/dotnet/sdk/blob/d86a7414a36eab675c7a92cba10c81c262a9c4e3/src/WebSdk/Web/Targets/Sdk.Server.targets#L29-L31

Publish gets imported after Web.ProjectSystem.